tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -193,6 +193,7 @@ def extension_modules():
     url='https://grpc.io',
     license='Apache License 2.0',
     classifiers=CLASSIFIERS,
+    long_description=open('README.rst').read(),
```

Comment:
Can we add [`long_description_content_type='text/x-rst'`](https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata) to indicate that the `long_description` is restructured text?

Reply:
Can we also explicitly close the file?

Reply:
Thanks for your advice.

```shell
$ twine check dist/*
Checking distribution dist/grpcio_tools-1.20.0.dev0-py2-none-any.whl: warning: `long_description_content_type` missing.  defaulting to `text/x-rst`.
Passed
Checking distribution dist/grpcio_tools-1.20.0.dev0-py3-none-any.whl: warning: `long_description_content_type` missing.  defaulting to `text/x-rst`.
Passed
```

My current solution is simple and working.

- The default `long_description_content_type` is `text/x-rst`.
- The file will be closed when this `setup.py` exited, which is not too long.

Of course, I like to make it better. But the original forked repository is deleted by accident, I have to send a new pull request #18289 to update.

Besides, `with` is not a good solution in this case. It will make the `setup.py` more complex. `setup.cfg` is a better solution, even when merging more files. For example:

```cfg
[metadata]
long_description = file: README.rst, CHANGELOG.rst
long_description_content_type = text/x-rst
```

My new pull request is something like that.